.github/workflows/requirements-locks.yml

```diff
@@ -0,0 +1,22 @@
+name: Test Requirements Lockfiles Check
+on:
+  push:
+    branches:
+      - 1.x
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  validate:
+    name: Check requirements lockfiles
+    runs-on: ubuntu-latest
+    container: ghcr.io/datadog/dd-trace-py/testrunner:latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Dependencies
+        run: pip install riot>=0.17.0
+
+      - name: Ensure riot requirements lockfiles are in sync
+        run: scripts/compile-and-prune-test-requirements && pwd && ls && git diff --exit-code
```

Comment:
hmm, it didn't seem to like `git diff --exit-code`... but that failed? how is it working in CircleCI?

> riot run -s snapshot-fmt && git diff --exit-code

🤔